style.css

```diff
@@ -19,7 +20,7 @@ a {
 }
 
 h1 {
-  font-size: 3rem;
+  font-size: 3.5rem;
```

Comment:
Can you explain the reasoning behind making the h1 bigger? Is it for legibility?
It would help us to review and approve this if your PR includes a more descriptive note regarding the intention of the changes you've made.